frontend/src/app/services/fullscreen.service.ts

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
The Guacamole sessions are not resized after changing to the full-screen mode. Large "black" borders appear in the session element: 

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/870f0ffa-71ec-4782-83eb-e8cd5eea5467)

This is similar behaviour like resizing the browser window, please have a look at: 

```py
  @HostListener('window:resize', ['$event'])
  onResize() {
    window.clearTimeout(this.debounceTimer);

    this.debounceTimer = window.setTimeout(() => {
      Array.from(document.getElementsByTagName('iframe')).forEach((iframe) => {
        const session = this.selectedSessions.find(
          (session) => 'session-' + session.id === iframe.id
        );

        if (session?.reloadToResize) {
          this.reloadIFrame(iframe);
        }
      });
    }, 250);
  }

```

Reply:
I only tested it with Jupyter sessions and forgot about resizing. Fortunately, including it was pretty simple because I only need to call `onResize()` when the fullscreen value changes.